src/Chapter3.hs

```diff
@@ -480,6 +522,54 @@ After defining the city, implement the following functions:
    and at least 10 living __people__ inside in all houses of the city totally.
 -}
 
+type Wall = Bool
+
+type Church = Bool
+type Library = Bool
+data Building = Library | Church
```

Comment:
You would have name clashes. It is better not to name the constructors as already defined types 🙂 

Btw, I don't see where you use those type aliases?